xplenty/xplenty_api.py

```diff
@@ -319,7 +340,7 @@ def terminate_cluster(self,id):
         return cluster
     
     
-    def create_cluster(self, cluster_type, nodes, cluster_name, cluster_description, terminate_on_idle=False, time_to_idle=3600):
+    def create_cluster(self, cluster_type, nodes, cluster_name, cluster_description, region, terminate_on_idle=False, time_to_idle=3600):
```

Comment:
@xomri - Please move region param to the end as this might break backward compatibility. Also, please make sure to give it a default value (for the same reasons).